app.js

```diff
@@ -1,19 +1,18 @@
 //jshint esversion:6
 
-require('dotenv').config();
-const express = require('express');
-const bodyParser = require('body-parser');
-const ejs = require('ejs');
-const gameRoutes = require('./routes');
+require("dotenv").config();
+const express = require("express");
+const bodyParser = require("body-parser");
+const ejs = require("ejs");
+const gameRoutes = require("./routes");
 
 const app = express();
 
-app.set('view engine', 'ejs');
-app.use(bodyParser.urlencoded({extended: true}));
-app.use(express.static(__dirname + '/public'));
+app.set("view engine", "ejs");
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.static(__dirname + "/public"));
 app.use(gameRoutes);
 
-app.listen(3000, function(){
-    console.log("Server is Running on Port 3000");
-
+app.listen(4000, function () {
```

Comment:
Did you change Port because of react using 3000 Port?

Reply:
I am again updating this and making a commit ....

Reply:
No issues, we will have to change that in future, just write new port number in console.log message